pkg/api/v1/types.go

```diff
@@ -115,6 +115,9 @@ type ObjectMeta struct {
 	// resource or set of resources. Only servers will generate resource versions.
 	ResourceVersion string `json:"resourceVersion,omitempty" description:"string that identifies the internal version of this object that can be used by clients to determine when objects have changed; populated by the system, read-only; value must be treated as opaque by clients and passed unmodified back to the server: http://docs.k8s.io/api-conventions.md#concurrency-control-and-consistency"`
 
+	// A sequence number representing a specific generation of the desired state.
+	Generation int64 `json:"generation,omitempty" description:"a sequence number representing a specific generation of the desired state."`
```

Comment:
I added it as a comment since whoever implements it for something else might forget to change the description. Did you mean the latter?